src/value.rs

```diff
@@ -18,6 +18,9 @@ use rust_decimal::Decimal;
 #[cfg(feature = "with-bigdecimal")]
 use bigdecimal::BigDecimal;
 
+#[cfg(feature = "with-uuid-0")]
+use uuid_0::Uuid as Uuid0;
```

Comment:
I proposed [this PR](https://github.com/SeaQL/sea-query/pull/343) not to have this behavior and keep `uuid` as a single and unique dependency.

Reply:
Ah -- Your PR is very similar to my original change when I first proposed my PR. The issue I ran into -- and you will see this when you remove the "draft" keyword and pre-commit testing is done -- is that you've now created mutually-exclusive features ([cargo docs about this](https://doc.rust-lang.org/cargo/reference/features.html#mutually-exclusive-features))

But we run all our tests with `cargo test --all-features`, and so you will see errors when that is run.

I had already suggested instead that we should have a "pecking order", where if UUID-1 is enabled it will automatically upstage UUID-0. I remember the issue being that sprinkling these conditional blocks everywhere gets kind-of nasty:

```rust
#[cfg(all(not(feature = "with-uuid-1"), feature = "with-uuid-0"))]
use uuid_0::Uuid as Uuid;

#[cfg(feature = "with-uuid-1")]
use uuid_1::Uuid as Uuid;

#[cfg(any(feature = "with-uuid-1", feature = "with-uuid-0"))]
// Do thing with Uuid

```

That being said, maybe now that @tyt2y3 has seen my PR and the juggling that has to be done to enable both at the same time, they may prefer that. But I think at-least you'll have to add a few more conditionals to implement this upstaging-behavior.

Reply:
I ran into this problem when trying to upgrade `time` in [this PR](https://github.com/SeaQL/sea-query/pull/341) and I'm trying to find a way to fix the CI.

Reply:
@ikrivosheev do you think we should be able to have `uuid@0.8` and `uuid@1` at the same time?

Reply:
On one hand: CI on the other hand - very strange behavior when we compile one library with a different version... 

I think it is better to panic when both features are enabled.